src/kata3.numberToReversedDigits.js

```diff
@@ -1,3 +1,10 @@
-const numberToReversedDigits = number => {};
+const numberToReversedDigits = (number) =>
```

Comment:
Brackets could be removed here
```suggestion
const numberToReversedDigits = number =>
```